tests/test_contact.py

```diff
@@ -0,0 +1,37 @@
+import jax
+import pytest
+
+import jaxsim.api as js
+from jaxsim import VelRepr
+
+
+def test_collidable_point_jacobians(
+    jaxsim_models_types: js.model.JaxSimModel,
+    velocity_representation: VelRepr,
+    prng_key: jax.Array,
+):
+
+    model = jaxsim_models_types
+
+    key, subkey = jax.random.split(prng_key, num=2)
+    data = js.data.random_model_data(
+        model=model, key=subkey, velocity_representation=velocity_representation
+    )
+
+    # =====
+    # Tests
+    # =====
+
+    # Compute the velocity of the collidable points with a RBDA.
+    # This function always returns the linear part of the mixed velocity of the
+    # implicit frame C corresponding to the collidable point.
+    W_ṗ_C = js.contact.collidable_point_velocities(model=model, data=data)
+
+    # Compute the generalized velocity and the free-floating Jacobian of the frame C.
+    ν = data.generalized_velocity()
+    CL_J_WC = js.contact.jacobian(model=model, data=data, output_vel_repr=VelRepr.Mixed)
```

Comment:
Is this correct? I was expecting it to be ${} ^{C[W]} J _{W, C}$ 🤔

Reply:
Yep you're right, that's the mixed velocity and I should have named the variable `CW_J_WC`. Thanks!